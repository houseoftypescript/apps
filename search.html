<!DOCTYPE html><html lang="en"><head><meta charSet="utf-8"/><title>Apps</title><meta name="description" content="Apps"/><meta name="viewport" content="width=device-width, initial-scale=1"/><link rel="icon" href="/favicon.ico"/><meta name="next-head-count" content="5"/><link rel="preload" href="/apps/_next/static/css/9c3c1c46d0f51455.css" as="style"/><link rel="stylesheet" href="/apps/_next/static/css/9c3c1c46d0f51455.css" data-n-g=""/><noscript data-n-css=""></noscript><script defer="" nomodule="" src="/apps/_next/static/chunks/polyfills-c67a75d1b6f99dc8.js"></script><script src="/apps/_next/static/chunks/webpack-6095028bfe210ee8.js" defer=""></script><script src="/apps/_next/static/chunks/framework-2c79e2a64abdb08b.js" defer=""></script><script src="/apps/_next/static/chunks/main-5020794388548d5e.js" defer=""></script><script src="/apps/_next/static/chunks/pages/_app-2fc2c01161ea9413.js" defer=""></script><script src="/apps/_next/static/chunks/639-566e8703ca81054f.js" defer=""></script><script src="/apps/_next/static/chunks/433-31ef9f38c7f6f138.js" defer=""></script><script src="/apps/_next/static/chunks/pages/search-9edc76579b4ec687.js" defer=""></script><script src="/apps/_next/static/rRAWw5qn3dPew_ppebK5K/_buildManifest.js" defer=""></script><script src="/apps/_next/static/rRAWw5qn3dPew_ppebK5K/_ssgManifest.js" defer=""></script></head><body><div id="__next"><div class="bg-fixed bg-center bg-cover text-white" style="background-image:url(/apps//_next/static/media/search.e9b854ad.jpeg)"><div class="min-h-screen bg-gray-900/75"><nav class="py-8"><style data-emotion="css 1qsxih2">.css-1qsxih2{width:100%;margin-left:auto;box-sizing:border-box;margin-right:auto;display:block;padding-left:16px;padding-right:16px;}@media (min-width:600px){.css-1qsxih2{padding-left:24px;padding-right:24px;}}@media (min-width:1200px){.css-1qsxih2{max-width:1200px;}}</style><div class="MuiContainer-root MuiContainer-maxWidthLg css-1qsxih2"><div class="flex justify-between text-white"><h1 class="text-2xl uppercase"><a href="/"><div class="flex items-center gap-2"><style data-emotion="css vubbuv">.css-vubbuv{-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;width:1em;height:1em;display:inline-block;fill:currentColor;-webkit-flex-shrink:0;-ms-flex-negative:0;flex-shrink:0;-webkit-transition:fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;font-size:1.5rem;}</style><svg class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-vubbuv" focusable="false" aria-hidden="true" viewBox="0 0 24 24" data-testid="SearchIcon"><path d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"></path></svg>HOM</div></a></h1><div><style data-emotion="css 11qrfta">.css-11qrfta{font-family:"Roboto","Helvetica","Arial",sans-serif;font-weight:500;font-size:0.875rem;line-height:1.75;letter-spacing:0.02857em;text-transform:uppercase;min-width:64px;padding:6px 8px;border-radius:4px;-webkit-transition:background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;color:#1976d2;}.css-11qrfta:hover{-webkit-text-decoration:none;text-decoration:none;background-color:rgba(25, 118, 210, 0.04);}@media (hover: none){.css-11qrfta:hover{background-color:transparent;}}.css-11qrfta.Mui-disabled{color:rgba(0, 0, 0, 0.26);}</style><style data-emotion="css 1ujsas3">.css-1ujsas3{display:-webkit-inline-box;display:-webkit-inline-flex;display:-ms-inline-flexbox;display:inline-flex;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;-webkit-box-pack:center;-ms-flex-pack:center;-webkit-justify-content:center;justify-content:center;position:relative;box-sizing:border-box;-webkit-tap-highlight-color:transparent;background-color:transparent;outline:0;border:0;margin:0;border-radius:0;padding:0;cursor:pointer;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;vertical-align:middle;-moz-appearance:none;-webkit-appearance:none;-webkit-text-decoration:none;text-decoration:none;color:inherit;font-family:"Roboto","Helvetica","Arial",sans-serif;font-weight:500;font-size:0.875rem;line-height:1.75;letter-spacing:0.02857em;text-transform:uppercase;min-width:64px;padding:6px 8px;border-radius:4px;-webkit-transition:background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;color:#1976d2;}.css-1ujsas3::-moz-focus-inner{border-style:none;}.css-1ujsas3.Mui-disabled{pointer-events:none;cursor:default;}@media print{.css-1ujsas3{-webkit-print-color-adjust:exact;color-adjust:exact;}}.css-1ujsas3:hover{-webkit-text-decoration:none;text-decoration:none;background-color:rgba(25, 118, 210, 0.04);}@media (hover: none){.css-1ujsas3:hover{background-color:transparent;}}.css-1ujsas3.Mui-disabled{color:rgba(0, 0, 0, 0.26);}</style><button class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium p-0 text-white css-1ujsas3" tabindex="0" type="button"><style data-emotion="css 6flbmm">.css-6flbmm{-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;width:1em;height:1em;display:inline-block;fill:currentColor;-webkit-flex-shrink:0;-ms-flex-negative:0;flex-shrink:0;-webkit-transition:fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;font-size:2.1875rem;}</style><svg class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge css-6flbmm" focusable="false" aria-hidden="true" viewBox="0 0 24 24" data-testid="MenuIcon"><path d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"></path></svg></button></div></div></div></nav><main class="pb-8"><style data-emotion="css 1qsxih2">.css-1qsxih2{width:100%;margin-left:auto;box-sizing:border-box;margin-right:auto;display:block;padding-left:16px;padding-right:16px;}@media (min-width:600px){.css-1qsxih2{padding-left:24px;padding-right:24px;}}@media (min-width:1200px){.css-1qsxih2{max-width:1200px;}}</style><div class="MuiContainer-root MuiContainer-maxWidthLg css-1qsxih2"></div></main></div></div></div><script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{}},"page":"/search","query":{},"buildId":"rRAWw5qn3dPew_ppebK5K","assetPrefix":"/apps","nextExport":true,"autoExport":true,"isFallback":false,"scriptLoader":[]}</script></body></html>