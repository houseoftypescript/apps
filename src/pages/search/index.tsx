import SearchTemplate from '@/templates/SearchTemplate';
import { NextPage } from 'next';

const SearchPage: NextPage = () => <SearchTemplate />;

export default SearchPage;
