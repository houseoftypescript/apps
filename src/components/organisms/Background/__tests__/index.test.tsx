import search from '@/assets/background/search.jpeg';
import { render } from '@testing-library/react';
import { Background } from '../index';

describe('Background', () => {
  it('to match snapshot', () => {
    const { container } = render(
      <Background backgroundImage={search}>
        <></>
      </Background>
    );
    expect(container).toMatchSnapshot();
  });
});
