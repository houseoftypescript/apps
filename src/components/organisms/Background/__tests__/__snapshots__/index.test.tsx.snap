// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Background to match snapshot 1`] = `
<div>
  <div
    class="bg-fixed bg-center bg-cover text-white"
    style="background-image: url(/img.jpg);"
  >
    <div
      class="transition-all"
      style="background-color: rgba(22, 25, 32, 0.5);"
    />
  </div>
</div>
`;
